superset-frontend/spec/javascripts/explore/components/SelectControl_spec.jsx

```diff
@@ -161,16 +130,6 @@ describe('SelectControl', () => {
         );
         expect(wrapper.html()).not.toContain('add something');
       });
-      it('shows numbers of options as a placeholder by default', () => {
```

Comment:
The number of options and remaining options has been removed as agreed with design and all the related tests have been removed as well